### PR TITLE
Added basic MongoDB Connection 

### PR DESCRIPTION
**Pull request by RichardHpa**
- Created env for MongoDB credentials (create your own or contact me for credentials)
- Basic connection for MongoDB
- Added README with instructions for setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+There is an example provided in the project
+
+### Install Dependencies & Run the Client & Server
+
+There are a few command you can run for this project. Depending on what you would like to do is which one you should use.
+
+```bash
+# Install dependencies for server
+npm run install:server
```
Comment (jenniferexong) [inline README.md:30]: Can we use yarn instead?

**Review by jenniferexong — APPROVED** [review]
Nice! Just wondering if we can/should use yarn instead of npm 🧐

File: README.md (hunk 1 above)
Comment (RichardHpa) [reply README.md:30]: We can if you want to. It doesn't really matter anymore as NPM has been upgraded quite a bit so its a a lot faster. Feel free to add some more commands if you want

Comment (jenniferexong) [reply README.md:30]: Alrighty all good with me!

**Pull request merged**